### PR TITLE
Update credits and dependancies

### PR DESCRIPTION
**Pull request by superpowers04**
e

### CODE REVIEW

File: src/main/resources/fabric.mod.json
Comment (skyrina) [inline src/main/resources/fabric.mod.json:1]: also also modify the discord invite link to https://discord.gg/nbN8KZyhUq

File: src/main/java/org/moon/figura/lua/api/entity/EntityAPI.java
Comment (skyrina) [inline src/main/java/org/moon/figura/lua/api/entity/EntityAPI.java:1]: also remove the isHamburger api entirely please

File: src/main/java/org/moon/figura/FiguraMod.java
Comment (skyrina) [inline src/main/java/org/moon/figura/FiguraMod.java:1]: could you change KOTMGit to MoonlightGit here?

**Review by skyrina — CHANGES_REQUESTED** [review]
change KOTMGit to MoonlightGit, remove isHamburger api, modify discord invite to https://discord.gg/nbN8KZyhUq

**Review by skyrina — APPROVED** [review]

**Comment by skyrina** [comment]
Since there are two pull requests that are the same thing, I have googled "random number from 1 to 2" and it said 2, so I'm going to merge https://github.com/Moonlight-MC-Temp/Figura/pull/2.

**Comment by skyrina** [comment]
reopened by request

**Review by Kichura — COMMENTED** [review]
To be superseded in #4.

**Pull request closed without merging**